src/main/java/EigenValue.java

```diff
@@ -0,0 +1,59 @@
+public class EigenValue {
+
+    public static void main(String[] args) {
```

Comment:
This is violating the single responsibility principle. EigenValue has two: calculating the eigenvalue and handling application startup. These should be separate.